docs/language/functions/unflatten.md

```diff
@@ -6,7 +6,7 @@ record.
 ### Synopsis
 
 ```
-unflatten(val: [{key:[string],value:any}) -> record
+unflatten(val: [{key:string|[string],value:any}) -> record
```

Comment:
I don't know that we use `|` inside a Zed type like this anywhere else. Maybe use a union?
```suggestion
unflatten(val: [{key:(string,[string]),value:any}) -> record
```

Reply:
We do:
https://zed.brimdata.io/docs/language/functions/len/
https://zed.brimdata.io/docs/language/functions/bucket/
https://zed.brimdata.io/docs/language/functions/network_of/

I don't know about union since that to me implies that the value must be a union of type `(string|[string])` though I suppose it could.

Reply:
Those are all top-level lists of types, though. None use `|` inside another complex type.

That said, I don't feel strongly about this, so stick with `|`.